openrtb_ext/imp_appnexus.go

```diff
@@ -1,13 +1,17 @@
 package openrtb_ext
 
-import "encoding/json"
+import (
+	"encoding/json"
+
+	"github.com/prebid/prebid-server/util/typeutil"
+)
 
 // ExtImpAppnexus defines the contract for bidrequest.imp[i].ext.prebid.bidder.appnexus
 type ExtImpAppnexus struct {
-	LegacyPlacementId       int                     `json:"placementId"`
+	LegacyPlacementId       typeutil.StringInt      `json:"placementId"`
 	LegacyInvCode           string                  `json:"invCode"`
 	LegacyTrafficSourceCode string                  `json:"trafficSourceCode"`
-	PlacementId             int                     `json:"placement_id"`
```

Comment:
should we use  `json.Number` type here? It handles both int and string values

Reply:
@onkarvhanumante We are not taking this option. Please see comment here - https://github.com/prebid/prebid-server/pull/2617#pullrequestreview-1322054803